src/Equinor.ProCoSys.IPO.Command/InvitationCommands/EditInvitation/EditInvitationCommandHandler.cs

```diff
@@ -96,7 +107,14 @@ public async Task<Result<string>> Handle(EditInvitationCommand request, Cancella
             }
             catch (Exception e)
             {
-                throw new Exception("Error: Could not update outlook meeting.", e);
+                if (await InvitationHelper.HasIpoAdminPrivilege(_permissionCache, _plantProvider, _currentUserProvider))
```

Comment:
I find this very strange. Will an exception be throw from Fusion.Integration.Meeting when an enduser use his Admin rights in IPO, and try to edit a meeting created by other than self?

Reply:
Yes. But we have decided that it is ok for an admin to edit the IPO even if the outlook meeting is not updated accordingly.